c-dependencies/js-compute-runtime/builtins/request-response.cpp

```diff
@@ -1929,6 +1949,24 @@ JSObject *Request::create(JSContext *cx, JS::HandleObject requestInstance, JS::H
         JS::GetReservedSlot(input_request, static_cast<uint32_t>(Slots::CacheOverride)));
   }
 
+  if (fastly_val.isObject()) {
+    JS::RootedValue decompress_response_val(cx);
+    JS::RootedObject fastly(cx, fastly_val.toObjectOrNull());
```

Comment:
I think this should just be `toObject()`. Note that `isObject()` in Spidermonkey API does not include null.

Reply:
We can't toObject here because we are in the embedded API which does not have a concrete definition of JSObject which we can import. We are guaranteed to return a JSObject pointer though, due to the isObject call above.

Reply:
Ah right, of course there are API differences!